packages/app/src/contact-support/helpers/downloading-logs.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { formatDate } from "Renderer/utils/format-date"
+import { ArchiveFile } from "App/main/functions/register-archive-files-listener"
+import getAppLogs from "Renderer/requests/get-app-logs.request"
+import downloadDeviceFile from "Renderer/requests/download-device-file.request"
+import getDeviceLogFiles from "Renderer/requests/get-device-log-files.request"
+import { DiagnosticsFilePath } from "@mudita/pure"
+
+export const todayFormatDate = formatDate(new Date())
+export const attachedFileName = `${todayFormatDate}.zip`
+
+export const downloadingLogs = async () => {
```

Comment:
🟡
- Return Type is missing
- No test

Reply:
downloadingLogs test and return type in [9f185d4](https://github.com/mudita/mudita-center/pull/586/commits/9f185d4bd7b2891cecf5832c9dbe6ecf43e73203)